logging/bin/getlogs.py

```diff
@@ -9,22 +9,34 @@
 import os, sys
 import json, csv
 
-## Cleanup temp file if not previously deleted
-if os.path.isfile("temp_SAS_OSQ.TXT"):
-    os.remove("temp_SAS_OSQ.TXT")
+## Cleanup temp file if not previously deleted, Create target directory
+if not os.path.isdir("GetLogs_Files"):
```

Comment:
In the future lets make an update to make these file names a variable, so we will only have to update them in one place if need be and will avoid mispellings that can cause issues.